GSA-TTS/federal-website-standards
# Replace the image on the banner standard page with the USWDS banner component

## Context
On the banner standard page we were showing an image to represent the banner component. This updates the banner standard page to put the actual USWDS banner component. Fixes #171 

## How to verify this change
1. [Visit the banner standard page](https://federalist-142078a8-f654-4daa-8f73-db9770b3ec7a.sites.pages.cloud.gov/preview/gsa-tts/federal-website-standards/caley/replace-images-with-uswds-components/standards/banner/) and observe that the banner under the "Example" heading is no longer an image but instead it's the actual component and you can expand/collapse the banner without it interfering with the main banner at the top of the page.
